docs/_static/link_gen/link.js

```diff
@@ -154,6 +165,7 @@ function displayLink() {
                 hubUrl, urlPath, repoUrl, branch
             );
         } else if (activeTab === "tab-auth-binder"){
+            var userName = new URL(repoUrl).pathname.split('/')[1];
```

Comment:
```suggestion
            // FIXME: userName parsing using new URL(...) assumes a 
            // HTTP based repoUrl. Does it make sense to create a
            // BinderHub link for SSH URLs? Then let's fix this parsing.
            var userName = new URL(repoUrl).pathname.split('/')[1];
```

@yuvipanda I'm not sure what to do here because I don't know what makes sense, but perhaps we can drop this comment or similarly.

Reply:
I love adding FIXMEs! Thanks, @consideRatio!